CenterForOpenScience/ember-osf-web
# [PRODUCT-907] clean up support page links

## Purpose

#610 made some changes to support page text, but also introduced and revealed some rogue spaces before periods because of `OsfLink`. This can be remedied by moving the links into the translations, which is a better place anyways because it allows whole-sentence translation.

I also noticed a couple of other inconsistencies with links, which I verified with product as ok to change:
* Frequently asked questions section title linked to `/faq`, while the link in the paragraph and the button link to `http://help.osf.io/m/faqs/l/726460-faqs`
* The OSF guides links all opened in a new tab, while no other links on the page did so (including the FAQ links that link to a subsection of the guides)

## Summary of Changes

* moved all links in paragraphs in to translations.
* changed Frequently asked questions section title to link to `faqPageUrl`
* removed `target="_blank"` from guides links

## Side Effects

n/a

## Feature Flags

n/a

## QA Notes

Make sure all links on the support page go to the correct places and have analytics.

## Ticket

https://openscience.atlassian.net/browse/PRODUCT-907

# Reviewer Checklist
